### ISSUE TITLE
[support] app store submission bug

### ISSUE THREAD

**Issue by sujithv28** [watson-developer-cloud/ios-sdk#239]
I get this error when trying to publish my app using application loader on iOS 9.3 with the latest Xcode

<img width="700" alt="screen shot 2016-05-01 at 6 12 31 pm" src="https://cloud.githubusercontent.com/assets/5770052/14944666/761e7644-0fc8-11e6-9c5b-df42c4735c2b.png">

**Comment by glennrfisher** [comment]
Hi @sujithv28. The discussion in #208 should help to resolve this problem. Let us know if that doesn't work.

**Comment by sujithv28** [comment]
Hi, was the fix to to remove the 'remove-copy-frameworks' branch? If so, how would we go about doing that?

**Comment by glennrfisher** [comment]
@sujithv28: The `remove-copy-frameworks` branch has been merged into the `master` branch, so no need to do anything there.

I would recommend starting with the [Carthage Readme](https://github.com/Carthage/Carthage#if-youre-building-for-ios-tvos-or-watchos). In particular, see the section on "If you're building for iOS, tvOS, or watchOS" which describes the use of a `copy-frameworks` script to work around an App Store submission bug.

I suspect the Carthage Readme will help solve your problem. If you still run into issues, though, these links proved helpful in #208:

- http://ikennd.ac/blog/2015/02/stripping-unwanted-architectures-from-dynamic-libraries-in-xcode/
- http://stackoverflow.com/questions/35240330/errors-building-xcode-project-after-adding-in-run-script-fatal-error-lipo-inpu

**Comment by sujithv28** [comment]
<img width="581" alt="screen shot 2016-05-04 at 7 39 10 pm" src="https://cloud.githubusercontent.com/assets/5770052/15032542/0387ce86-1230-11e6-87af-e59836333f3b.png">

@glennrfisher I managed to get a lot of the errors down but now I only have these errors left and am not sure how to fix them.

**Comment by glennrfisher** [comment]
Okay, I found the problem this morning. We made those changes and merged them into the `master` branch, but didn't issue a release. So Carthage wasn't pulling them into your local copy.

Try running `carthage update --platform iOS` to update to the `v0.2.5` release. Hopefully that fixes the two remaining errors you see when uploading to the app store.

**Comment by glennrfisher** [comment]
Please let us know if this fixes the issue after you get a chance to try it out! Otherwise we'll keep investigating and see what we can dig up.

**Comment by sujithv28** [comment]
@glennrfisher I seemed to have gotten the error again after pulling the latest release but I'll double check that the update succeeded. I also have an issue where if I do `carthage update --platform iOS --no-use-binaries`, it gets stuck on fetching XCGLogger and takes up a lot of the CPU usage.

**Comment by glennrfisher** [comment]
Hmm, that's odd. We removed the XCGLogger dependency a couple months ago. I double-checked the release, and it is not referenced in the `Cartfile` or `Cartfile.resolved` for the SDK.

I'm not sure what's causing these issues, but here's a couple of things to try:

1. Reinstall the latest version of Carthage by deleting `/Library/Frameworks/CarthageKit.framework` and then using Homebrew to run `brew update && brew install carthage`.
2. Trash your app's `Carthage` directory and `Cartfile.resolved` to force Carthage to re-clone all of the dependencies.

**Comment by sujithv28** [comment]
Okay so I got it working and here is what I did. I deleted the 'CarthageKit.framework' file and reinstalled the latest version of 'Carthage'. I also had to relink it by removing another file. I then removed the 'Carthage directory' and the 'Carthage.resolved' file and reinstalled all the dependencies. After that I added a run script from http://ikennd.ac/blog/2015/02/stripping-unwanted-architectures-from-dynamic-libraries-in-xcode/ and rebuilt the project and submitted it.

Thanks @glennrfisher for helping me get it working
